component/azstorage/block_blob_test.go

```diff
@@ -3156,6 +3156,106 @@ func (s *blockBlobTestSuite) TestInvalidMD5OnReadNoVaildate() {
 	}
 }
 
+func (s *blockBlobTestSuite) TestDownloadBlobWithCPKEnabled() {
+	defer s.cleanupTest()
+	s.tearDownTestHelper(false)
+
+	config := fmt.Sprintf("azstorage:\n  account-name: %s\n  endpoint: https://%s.blob.core.windows.net/\n  type: block\n  account-key: %s\n  mode: key\n  container: %s\n  cpk-enabled: true\n  cpk-encryption-key: %s\n  cpk-encryption-key-sha256: %s\n",
```

Comment:
what kind of values you added to your json file for this. Do we need to generate them on the storage account we test against. If you can share the details on how to configure this I can go ahead and configure. Getting pipeline access will not solve this as these accounts are statically configured and all our pipelines run against them. Pipeline access is only allowed to blobfuse dev team as of now.

Reply:
Our pipelines have access to storage accounts in our subscription so for security reasons we do not allow access to pipelines to anyone outside the team.

Reply:
'cpk-encryption-key' and 'cpk-encryption-key-sha256' are the two values i used in json. I think modifying the test itself should be easy then. Let me know if u think otherwise?

Reply:
As discussed offline, going ahead with generating key and sha in the tests itself.